### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Status

Ready for review

## Description of Changes

* Description: 
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/securedrop-dev-docs/commit/fcb4b2bcf56c3b4943d2cff3b6cec30c344a4f5c#commitcomment-89069829

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/securedrop-dev-docs/commit/174e598e5e8ed7d2544569f189e990b9bae9a440

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.
* Resolves #
 

* Related Issues

## Testing

*

## Release 

* 

## Checklist (Optional)

- [ ] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [ ] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/dockerbuildmaint.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Build container
 ===============
 We use a Docker build container to build our debian packages for SecureDrop (via ``make build-debs``
-in the ``securedrop`` Github repository root directory). We keep images of this our container in a
+in the ``securedrop`` GitHub repository root directory). We keep images of this our container in a
```
Comment (jsoref) [inline docs/dockerbuildmaint.rst:6]: Brand

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine.

**Review by legoktm — DISMISSED** [review]
Thanks, all of these look great to me. Could you squash it down to one commit? Or I can take care of that for you.

**Comment by jsoref** [comment]
Squashing is a configurable property of repositories...
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request

It needs to be enabled:
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-merging-for-pull-requests

If it isn't enabled (and you can't enable it), I'm happy to squash...

**Comment by gonzalo-bulnes** [comment]
Oh, thanks @jsoref, but that's not what I meant. I personally don't like much how GitHub squashes branches automatically. It might actually be fine in the case of this branch, but most of the time I prefer grouping commits in a more mindful way than what the automated options allow. So I wouldn't advocate for enabling any of those options.

We usually tidy up each others branches when they belong to the `freedomofpress` remote. I don't know any way to do that on anyone's personal forks, without affecting the corresponding PR. That's where I thought there might be a way I haven't thought of. :slightly_smiling_face:

**Comment by gonzalo-bulnes** [comment]
If you squash the commits, I'm happy to merge this :+1:

**Comment by jsoref** [comment]
Ah, GitHub only lets you interact w/ someone's fork via the PR (if the forker lets you). In general, if a forker has allowed maintainers to touch their branch, they've allowed them to do that, and it's ok. 

For this PR, I've enabled that feature:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Depending on your repo policies, you might be able to push directly and include a `closes #...` notation.

Anyway, I've squashed. Personally I prefer letting repository maintainers do it, because they're more likely to have opinions about what the commit messages should say than I do 🙂.

**Comment by gonzalo-bulnes** [comment]
@eloquence Something seems off. There don't seem to be many CI pipelines that have been triggered form a PR recently in this repo. The checks are required though. :thought_balloon:

**Comment by eloquence** [comment]
The repo was not configured to build from forks, sorry about that. This should now be fixed -- if either of you push a commit or amend, it _should_ trigger a build now.

**Pull request merged**